### PR TITLE
Sector encoding

### PR DESCRIPTION
**Pull request by RemoteSalad**
Added 3 methods primarily for encoding the sectors.

The second two methods can be used for other columns as well.

The sector groups in the first method are of my own creation so feel free to change them.

I wrote a few tools to help me when trying to reason through different groupings and see how it affected their relative sizes. They aren't included here but If you want to explore different groupings I can provide them.

### CODE REVIEW

**Review by sneaky-hobbitses — APPROVED** [review]
Great work. Just this part:
`make_sector_group_column(df, **give_group_profiles = True**):`

File: data_scientist_skills/data_engineering.py [hunk 1]
```diff
@@ -76,3 +80,106 @@ def process_salary_estimate(dataframe, only_mean = True):
     temp_df.drop(columns = 'salary_estimate', inplace = True)
 
     return temp_df
+
+
+###Sector Encoding Section
+
+def make_sector_group_column(df, give_group_profiles = False):
```
Comment (RemoteSalad) [inline data_scientist_skills/data_engineering.py:186]: The second parameter isn't used actually/giving group profiles isn't implemented here. I have the code but I thought it better to leave it out unless desired

Comment (RemoteSalad) [inline data_scientist_skills/data_engineering.py:186]: The second parameter isn't used actually/giving group profiles isn't implemented here. I have the code but I thought it better to leave it out unless desired

File: data_scientist_skills/data_engineering.py [hunk 2]
```diff
@@ -76,3 +80,106 @@ def process_salary_estimate(dataframe, only_mean = True):
...
+
+    [ 'Telecommunications', 'Media','Real Estate','Travel & Tourism',
+     'Arts, Entertainment & Recreation','Restaurants, Bars & Food Services','Retail',
+     'Non-Profit','Government','Education','Consumer Services',],
+
+    ['Health Care','Biotech & Pharmaceuticals',],
+
+    ['Information Technology',],
+
+    ['Business Services',]
```
Comment (honnsey) [inline data_scientist_skills/data_engineering.py:219]: why do we need the comma at the end of a list?

Comment (RemoteSalad) [reply data_scientist_skills/data_engineering.py:219]: No need, it's just a result of trying different grouping combinations.

I left them for ease in trying different groupings.

**Review by sneaky-hobbitses — APPROVED** [review]

**Review by sneaky-hobbitses — APPROVED** [review]

**Review by honnsey — APPROVED** [review]

**Pull request merged**